README.md

```diff
@@ -20,7 +20,7 @@
 
 | Version | Build Status | Code Coverage |
 |:---------:|:-------------:|:-----:|
-| `2.x`| [![Build Status][Travis 2.x Build Status Image]][Travis Build Status] [![Build Status][AppVeyor 2.x Build Status Image]][AppVeyor Build Status] | [![Coverage Status][2.x Code Coverage Image]][2.x Code Coverage] |
+| `2.x`| ![CI](https://github.com/getsentry/sentry-php/workflows/CI/badge.svg?branch=2.x) ![Static analysis](https://github.com/getsentry/sentry-php/workflows/Code%20style%20and%20static%20analysis/badge.svg?branch=2.x) | [![Coverage Status][2.x Code Coverage Image]][2.x Code Coverage] |
```

Comment:
There are still references to the old CI badges/links at the end of this file. Either remove them or fix them to point to the GitHub Actions. Also, does it makes sense to show the status of the static analysis workflow?

Reply:
It's still part of the build. Having it split it two files means that we have two separate status.
We could show the first one only, right.